main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -70,17 +70,14 @@ class LanguageServer(port: Int, o: Options) extends WebSocketServer(new InetSock
     * The custom date format to use for logging.
     */
   val DateFormat: String = "yyyy-MM-dd HH:mm:ss"
-
-  /**
-    * The Flix instance (the same instance is used for incremental compilation).
-    */
-  private val flix: Flix = new Flix().setFormatter(NoFormatter).setOptions(o)
-
   /**
     * A map from source URIs to source code.
     */
   val sources: mutable.Map[String, String] = mutable.Map.empty
-
+  /**
+    * The Flix instance (the same instance is used for incremental compilation).
+    */
+  private val flix: Flix = new Flix().setFormatter(NoFormatter).setOptions(o)
```

Comment:
I still see some unnecessary changes to this file. I think it might be best to checkout the original file and then port your changes to it.

Reply:
The reason I care is because I don't want to break "git blame" etc.

Reply:
Sorry, I somehow missed these 🤔